### PR TITLE
Added updates to Push/Pull episode

### PR DESCRIPTION
**Pull request by anelda**

### CODE REVIEW

**Comment by anelda** [comment]
Oops sorry! I never created a new branch for the first PR that's why everything is rolled into one

File: _episodes/04-changes.md [hunk 1]
```diff
@@ -156,11 +156,19 @@ $ git status
...
 ~~~
 {: .output}
 
-it tells us everything is up to date.
+it tells us that our local repository is now differing from the
+origin (the repository from which we cloned this one, i.e. the original repository in Github)
+by one commit.
+
+It also tells us that we can use the `git push` command to send our commit to the original repo in 
+Github to syncronise these two repositories.
```
Comment (biologyguy) [inline _episodes/04-changes.md]: syncronise -> synchronize

File: _episodes/05-push-pull.md [hunk 2]
```diff
@@ -60,7 +63,8 @@ To https://github.com/biologyguy/conversions.git
 
 &nbsp;
 
-Now all of the changes have been uploaded to GitHub:
+If you refresh your browswer showing your Github repository created earlier in the lesson, it will
```
Comment (biologyguy) [inline _episodes/05-push-pull.md]: browswer -> browser

**Review by biologyguy — COMMENTED** [review]
Looks good! Just a couple minor typos, then I'll merge.

**Pull request merged**